packages/backend-defaults/src/types.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AnyServiceFactory } from '@backstage/backend-plugin-api';
+
+/**
+ * @public
+ */
+export interface CreateBackendOptions {
+  serviceFactories?: AnyServiceFactory[];
```

Comment:
Should we change this naming to `services` perhaps? Even though this name may be technically true, it's a detail and the reality is hidden behind our usages of `x` vs `x()` and `x({ option: true })` and whatnot other curried tomfoolery, and doesn't really add value to the reader methinks

Reply:
Yes that makes sense 👍